xc/xc7/CMakeLists.txt

```diff
@@ -1,9 +1,14 @@
 set(XC7_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(PRJXRAY_DIR ${symbiflow-arch-defs_SOURCE_DIR}/third_party/prjxray
   CACHE PATH "Path to prjxray library")
-set(PRJXRAY_DB_DIR
-  ${symbiflow-arch-defs_SOURCE_DIR}/third_party/prjxray-db
-  CACHE PATH "Path to prjxray database files")
+
+execute_process(
+  COMMAND
+    prjxray-config
```

Comment:
You should use `find_program` to locate `prjxray-config`.   This causes CMake to report if `prjxray-config`  is missing from the path

Reply:
Make sure to specific the "DOC <description>" argument, and REQUIRED argument.  See https://cmake.org/cmake/help/latest/command/find_program.html for details.

Reply:
I have actually added prjxray-config with the `add_env_executable` function

Reply:
Done